lib/diagnostic.rb

```diff
@@ -108,10 +111,11 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = {:favorite_number, :first_name}
 
```

Comment:
Close! You just forgot to instantiate the hash with the values
`person = {first_name: "Lauren", favorite_number: 7}`